src/Workspaces/Core/Portable/Workspace/DocumentSpan.cs

```diff
@@ -2,9 +2,6 @@
 
```

Comment:
does moving this affect TS/JS?

Reply:
good question. let me put type forwarding.

Reply:
actually I moved the type back to original location.